### ISSUE TITLE
liquid chromatography

### ISSUE THREAD

**Issue by chung88** [BioAssayOntology/BAO#106]
BAO I see http://purl.obolibrary.org/obo/ERO_0001500, but in OBO shows as a BAO term:
https://ontobee.org/search?ontology=&keywords=liquid+chromatography+&submit=Search+terms
http://purl.obolibrary.org/obo/ERO_0001500 (ERO):
liquid chromatography in Ontobee: [BAO](https://ontobee.org/ontology/BAO?iri=http://purl.obolibrary.org/obo/ERO_0001500)

subClassOf http://purl.obolibrary.org/obo/ERO_0001500 fails
BAO_0040042	subClassOf	rdf_property	BAO_0040031

**Comment by JoanGlennyPescov** [comment]
Yes but there is not BAO ID for it in OBO. I would probably just create a new BAO ID for it.

Terms with 'ERO_0001500' included in their label:

http://purl.obolibrary.org/obo/ERO_0001500 (ERO):
liquid chromatography in Ontobee: [BAO]

**Comment by chung88** [comment]
Please understand the complexity of the steps for QA/QC.  This means: 

1. delete http://purl.obolibrary.org/obo/ERO_0001500
2. create a BAO term (provide parent, definition and source)
3. update the axioms

See below:
- OBO term liquid chromatography (http://purl.obolibrary.org/obo/ERO_0001500) has been removed from the ontology

NCBI BioPortal https://bioportal.bioontology.org/ontologies/ERO/?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FERO_0001500&jump_to_nav=true  
- The last maintenance date of ERO was 2013
- BAO the term  liquid chromatography has 33 axioms
Row      label property    value
1     Tandem Mass Tag (TMT) MS Protein Quantification BAO_0095009 http://purl.obolibrary.org/obo/ERO_0002176
2     Liquid Chromatography/Mass Spectroscopy (LC/MS) protein quantification  BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
3     metabolite identification and profiling assay   BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
4     hepatocyte low clearance relay assay      BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
5     simulated intestinal fluid stability assay      BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
6     MAO reaction phenotyping assay      BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
7     blood to plasma ratio assay   BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
8     placental transfer assay      BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
9     mass balance assay      BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
10    biliary excretion assay BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
11    renal excretion assay   BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
12    CYP3A4 induction assay  BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
13    BCRP inhibition assay   BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
14    MRP2 inhibition assay   BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
15    MATE1 inhibition assay  BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
16    OAT1 inhibition assay   BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
17    OAT2 inhibition assay   BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
18    OAT3 inhibition assay   BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
19    OATP1B1 inhibition assay      BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
20    OATP1B3 inhibition assay      BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
21    OCT1 inhibition assay   BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
22    MRP2 substrate assay    BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
23    BCRP substrate assay    BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
24    MATE1 substrate assay   BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
25    OAT1 substrate assay    BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
26    OAT2 substrate assay    BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
27    OAT3 substrate assay    BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
28    OATP1B1 substrate assay BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
29    OATP1B3 substrate assay BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
30    OCT1 substrate assay    BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
31    OATP2B1 substrate assay BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
32    monoamine oxidase inhibition assay  BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
33    human mammary epithelial cell permeability assay      BAO_0095009 http://purl.obolibrary.org/obo/ERO_0001500
